canonical-web-and-design/jaas-dashboard
# Add a provider icon to the model list

## Done
Add known provider icon to the model list. Supporting "aws", "gce", "azure" and "kubernetes".

## QA
- Open the demo
- Check the provider type icons you see on the model list

## Details
Fixes https://github.com/canonical-web-and-design/jaas-dashboard/issues/954

## Screenshots
![image](https://user-images.githubusercontent.com/1413534/114617125-9cd6fb80-9c9f-11eb-9adc-7004a7f1941a.png)
